mvpa2/generators/partition.py

```diff
@@ -491,6 +493,54 @@ def _get_partition_specs(self, uniqueattrs):
                     if ind < self.count]
 
 
+class FactorialPartitioner(Partitioner):
+    """
+    Given another partitioner on a dataset with two hierarchically organized
+    attributes, generates the combinations of all possible folds of the
+    superordinate attribute with the balanced combinations of the subordinate
+    attribute. For example, splitting familiar and unfamiliar faces while
+    balancing identities in the training/test set.
+
+    ADD EXAMPLE HERE
+    """
+    def __init__(self, partitioner, **kwargs):
+        Partitioner.__init__(self, **kwargs)
```

Comment:
to be safe, use `super` construct (even if other Partitioners use old fashion way like this, don't remember) -- we do have multiple inheritence around the hierarchy